native-image/what-the-dickens/README.md

```diff
@@ -206,161 +157,134 @@ This means it will run as a part of the `package` phase.
+1. Use the command below to launch the Spring application with the Native Image tracing agent attached.
 
-![](images/RMIL_Technology_Laptop_Bark_RGB_50.png#input)
-```bash
-./mvnw -Pnative -DskipTests=true clean package
-```
+    ```bash
+    java -Dpring.aot.enabled=true \
+    -agentlib:native-image-agent=config-output-dir=src/main/resources/META-INF/native-image/ \
+    -jar target/what_the_dickens-0.0.1-SNAPSHOT.jar
+    ```
 
-> Note: Depending on your platform, this can take several minutes.
+2. Stop the application with Ctrl-C.
 
-This will generate a native executable in the _target_ directory, named _What\_the\_Dickens_.
+    On application shutdown, the tracing agent writes the configuration file _reachability-metadata.json_ to the specified output directory: _src/main/resources/META-INF/native-image/_.
+    In this case, it's the default directory that Native Image expects to find the configuration file.
 
-Run this native executable in the background:
+3. Now run the Maven build using the profile, as below (note that the profile name is specified with the `-P` option, and you can skip the tests from now on):
 
-![](images/RMIL_Technology_Laptop_Bark_RGB_50.png#input)
-```bash
-./target/What_the_Dickens &
-```
+    ```bash
+    ./mvnw -Pnative -DskipTests=true package
+    ```
+    This generates a native executable in the _target_ directory, named _what\_the\_dickens_.
 
-When the application starts, you should see something similar to the following:
+4. Run this native executable in the background:
 
-```
-Started DickensApplication in 12.829 seconds (process running for 12.925)
-```
+    ```bash
+    ./target/what_the_dickens &
+    ```
+    When the application starts, you should see something similar to the following:
+    ```
+    Started DickensApplication in 2.86 seconds (process running for ...)
+    ```
 
-Test it, using the following commands:
+5. Test it, using the following commands:
 
-![](images/RMIL_Technology_Laptop_Bark_RGB_50.png#input)
-```bash
-curl http://localhost:8080/What_the_Dickens
-curl http://localhost:8080/What_the_Dickens/30
-```
+    ```bash
+    curl http://localhost:8080/whatTheDickens
+    curl http://localhost:8080/whatTheDickens/30
+    ```
 
-Now you have a native executable of your application--however, it starts no faster than the original Java version.
-This is mainly because the performance is constrained by the amount of time the application takes to ingest the Dickens' novels. 
+    Now you have a native executable of your application--however, it starts no faster than the original Java version.
+    This is mainly because the performance is constrained by the amount of time the application takes to ingest the Dickens' novels.
 
-Stop the application before you move on, using Ctrl-C.
+6. Stop the application before you move on as before (type `fg` and then Ctrl-C).
 
 ## **STEP 3**: Reducing the Startup Time of Your Native Executable
```

Comment:
This step doesn't work ( and it isn't being tested with GitHub workflow). 
Since the intialization strategy evolved in Native Image since JDK 17, `--initialize-at-build-time=com.example.DickensController,rita.RiMarkov,rita.Util,rita.Tokenizer,rita.RiTa` causes the build to fail. I need to spend more time on this step. Or we jjust remove it all together and don't promote class initialization options.